soc/arm64/intel_socfpga/agilex5/Kconfig.defconfig.agilex5

```diff
@@ -10,7 +10,7 @@ config SOC
 # - include the UART interrupts 173 or 204
 config NUM_IRQS
 	int
-	default 205
+	default 274
```

Comment:
why this change? (commit message could tell)

Reply:
Initially, the maximum IRQ number did not encompass certain IRQ numbers. The Agilex5 SoC FPGA device supports a maximum IRQ of 274. Specifically, the XGMAC IRQ exceeds 205. Therefore, during the development of this driver, I made sure to update it accordingly.